packages/utils/src/envelope.ts

```diff
@@ -187,6 +187,11 @@ export function createSpanEnvelopeItem(spanJson: Partial<SpanJSON>): SpanItem {
     type: 'span',
   };
 
+  // ensure we have exclusive_time
+  if (typeof spanJson.exclusive_time === 'undefined' && spanJson.timestamp && spanJson.start_timestamp) {
+    spanJson.exclusive_time = spanJson.timestamp - spanJson.start_timestamp;
+  }
```

Comment:
Should we add a comment stating that this is not actual "exclusive time" - maybe a TODO stating that this should actually be calculated?

Reply:
ah right woops